app/assets/javascripts/components/AccountMenu/Encryption.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+
+type Props = {
+  isEncryptionEnabled: boolean;
+  notesAndTagsCount: number;
+  encryptionStatusString: string | undefined;
```

Comment:
```suggestion
  encryptionStatusString?: string;
```

Reply:
Although in this case there won't be a difference, but having `string | undefined` enforces us to pass `encryptionStatusString` to `Encryption` component, while with `encryptionStatusString?` we can skip that prop and TS won't complain. The point is we expect `encryptionStatusString` to be passed, it just can be either `string` or `undefined`.

Reply:
That's a really good point! 🙂